src/solver/conv_bin_winoRxS.cpp

```diff
@@ -246,6 +248,11 @@ bool ConvBinWinogradRxS::IsApplicable(const ConvolutionContext& params) const
 
     const auto name = params.GetStream().GetDeviceName();
     const bool fp16 = params.IsFp16();
+
+#if WORKAROUND_ISSUE_1146
```

Comment:
Similar to previous comments

Reply:
@junliume This Solver is enabled on gfx908. We are going to to enable it on gfx90a as well. Even if technically is not necessary, I would better keep it here as a reminder.

@Slimakanzer [Recommendation] Rework this code in order to make the W/A really necessary ;)

Reply:
@atamazov 
Thank you! Reverted this unnecessary restriction (see [explanation](https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1146#issuecomment-919021192))